src/_content/posts/keynote-carson.md

```diff
@@ -0,0 +1,69 @@
+---
+author:  Communications Team
+category: General
+published_datetime: 2025-07-27 06:00:00-04:00 
+title: "Announcing Our Keynotes: Carson Gross"
+
+cover:
+    url: /assets/img/2025/Carson.png
+    alt: "Announcing Our Keynotes"
+---
+
+### We're honored to announce our first keynote speaker for DjangoCon US 2025: Carson Gross
+
+We had the opportunity to sit down with Carson and ask him a few questions about his work, his upcoming talk, and what he's looking forward to at DjangoCon US 2025.
+
+## How do you use Django?
+
+> I do not use Django very often because my main profession is as a teacher and we mainly use Java here. However, the Django community has embraced HTMX, a web library that I created, and so I have a lot of friends in the community.
```

Comment:
```suggestion
> I do not use Django very often because my main profession is teaching and we mainly use Java. However, the Django community has embraced HTMX, a web library that I created, and so I have a lot of friends in the community.
```